telegram-notifier/LambdaFunction.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+
+import boto3
+import json
+import logging
+import os
+import urllib
+from urllib2 import Request, urlopen, URLError, HTTPError
+
+"""
+BOT_ID = Your Telegram BOT ID. Example: bot123456789:XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+CHAT_ID = Is your channel ID or your BOT chat window id
+"""
+
+BOT_ID = "bot123456789:XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+CHAT_ID = "01234567891234"
+API_ENDPOINT = "https://api.telegram.org/%s/sendMessage" % BOT_ID
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+    message = str(event['detail']['eventDescription'][0]['latestDescription'] + "\n\n<https://phd.aws.amazon.com/phd/home?region=us-east-1#/event-log?eventID=" + event['detail']['eventArn'] + " | Click here> for details.")
+
+    msg = {
+        "chat_id":CHAT_ID,
+        "text": message
+    }
+
+    logger.info(str(msg))
+
+    req = Request(API_ENDPOINT, urllib.urlencode(msg))
```

Comment:
using the [requests module](http://docs.python-requests.org/en/master/) here will probably be more concise, and probably more familiar to python coders. 

as an added plus there's no need to bundle the lib in your deployment package as it's included in botocore:
```python
import botocore.vendored.requests as requests
```